base/commands/viridian/download_logs_it_test.go

```diff
@@ -21,24 +22,22 @@ func downloadLogs_NonInteractiveTest(t *testing.T) {
 			tcx.CLCExecute(ctx, "viridian", "download-logs", c.ID, "--output-dir", dir)
 			tcx.AssertStderrContains("OK")
 			require.FileExists(t, paths.Join(dir, "node-1.log"))
-			require.FileExists(t, paths.Join(dir, "node-2.log"))
-			require.FileExists(t, paths.Join(dir, "node-3.log"))
 		})
 	})
 }
 
 func downloadLogs_InteractiveTest(t *testing.T) {
 	viridianTester(t, func(ctx context.Context, tcx it.TestContext) {
 		dir := check.MustValue(os.MkdirTemp("", "log"))
-		defer func() { check.Must(os.RemoveAll(dir)) }()
+		t.Logf("Downloading to directory: %s", dir)
+		//defer func() { check.Must(os.RemoveAll(dir)) }()
 		tcx.WithShell(ctx, func(tcx it.TestContext) {
 			tcx.WithReset(func() {
 				c := createOrGetClusterWithState(ctx, tcx, "")
+				time.Sleep(10 * time.Second)
```

Comment:
Instead of putting sleep would it be better to get a cluster with state `RUNNING`? I guess that is the reason for sleep right?

Reply:
I agree, updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/232/commits/f8e7230dae1a5e5703035586a34996ccc93a436b